### PR TITLE
stream aggregation: fix type of command-line flag `-remoteWrite.strea…

### PR DESCRIPTION
**Pull request by Haleygo**
…mAggr.ignoreFirstIntervals`

, to accept multiple values which can be applied to the corresponding `-remoteWrite.url`

### CODE REVIEW

File: app/vmagent/remotewrite/streamaggr.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ var (
 	streamAggrIgnoreOldSamples = flagutil.NewArrayBool("remoteWrite.streamAggr.ignoreOldSamples", "Whether to ignore input samples with old timestamps outside the current "+
 		"aggregation interval for the corresponding -remoteWrite.streamAggr.config at the corresponding -remoteWrite.url. "+
 		"See https://docs.victoriametrics.com/stream-aggregation/#ignoring-old-samples")
-	streamAggrIgnoreFirstIntervals = flag.Int("remoteWrite.streamAggr.ignoreFirstIntervals", 0, "Number of aggregation intervals to skip after the start "+
+	streamAggrIgnoreFirstIntervals = flagutil.NewArrayInt("remoteWrite.streamAggr.ignoreFirstIntervals", 0, "Number of aggregation intervals to skip after the start "+
```
Comment (hagen1778) [inline app/vmagent/remotewrite/streamaggr.go:53]: This requires vmagent.md to be updated as well.

Comment (hagen1778) [reply app/vmagent/remotewrite/streamaggr.go:53]: Btw, this change helped me to notice that `streamAggrDropInputLabels ` is misused.

@Haleygo @AndrewChubatiuk would you mind fixing this as well?

**Review by hagen1778 — CHANGES_REQUESTED** [review]

File: app/vmagent/remotewrite/streamaggr.go (hunk 1 above)
Comment (AndrewChubatiuk) [reply app/vmagent/remotewrite/streamaggr.go:53]: do you mean this issue?
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6743

Comment (hagen1778) [reply app/vmagent/remotewrite/streamaggr.go:53]: Yes

Comment (AndrewChubatiuk) [reply app/vmagent/remotewrite/streamaggr.go:53]: there're two issues in this PR, the one, which is not fixed in it, requires changes in string array flag type

Comment (Haleygo) [reply app/vmagent/remotewrite/streamaggr.go:53]: Fixed docs.

**Review by hagen1778 — APPROVED** [review]
Thanks!

**Pull request merged**